plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/plugin/PluginUpdateManager.kt

```diff
@@ -128,7 +128,8 @@ class PluginUpdateManager : Disposable {
                         result = Result.Succeeded,
                         id = "autoUpdateActionRestart",
                         source = SOURCE_AUTO_UPDATE_FINISH_NOTIFY,
-                        component = Component.Filesystem
+                        component = Component.Filesystem,
+                        action = "autoUpdateActionRestart"
```

Comment:
@dhruvigajjar do you know if this is the intent of the action field?

Reply:
I don't know if this is the intend, but build is failing due to this new required action field post telemetry definition generation hence I have temporarily put the id as action. Let me know if this needs to be changed.

Reply:
This is when user clients chooses an action from notification actions notification provided. 
mostly to track if users dismiss or reauth from these notification.

Reply:
if these default action, we should use `toolkit_showNotication`